lib/compat/wordpress-6.1/template-parts-screen.php

```diff
@@ -187,7 +187,7 @@ static function( $classes ) {
 	wp_enqueue_media();
 
 	if (
-		current_theme_supports( 'wp-block-styles' ) ||
+		current_theme_supports( 'wp-block-styles' ) &&
```

Comment:
This isn't directly related to the issue this PR is trying to address, but I noticed this logic wasn't quite the same as what is used in `client-assets.php`.

Reply:
This needs to be updated in Core in [wp-admin/site-editor.php](https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-admin/site-editor.php#L148).